examples/SMIRNOFF_comparison/compare_set_energies.py

```diff
@@ -46,7 +48,6 @@
     from openforcefield.tests.utils import FailedParameterComparisonError, FailedEnergyComparisonError
     try:
         # We ignore the charges as they are not included in the force field.
-        # TODO: Reactivate this check when we'll be able to load charges from the file.
```

Comment:
My rationale for removing it was actually to make the code more "documentation-like", but I agree it might be premature to do so. I'll re-add, thanks!